metrics/vendor/github.com/red-hat-storage/ocs-operator/api/v4/v1/storagecluster_types.go

```diff
@@ -569,6 +569,9 @@ type StorageClusterStatus struct {
 	// Images holds the image reconcile status for all images reconciled by the operator
 	Images ImagesStatus `json:"images,omitempty"`
 
+	// CephDeviceClasses holds the device classes available in the ceph cluster
+	CephDeviceClasses []string `json:"cephDeviceClasses,omitempty"`
```

Comment:
Per previous comment, how about this?
```suggestion
	DefaultDeviceClass string `json:"defaultDeviceClass,omitempty"`
```